lib/manager/regex/index.ts

```diff
@@ -47,7 +47,12 @@ function createDependency(
     const fieldTemplate = `${field}Template`;
     if (config[fieldTemplate]) {
       try {
-        dependency[field] = template.compile(config[fieldTemplate], groups);
+        const filterFields = false;
```

Comment:
Why extra variable?

Reply:
The input to this compilation is the matching regex capture groups, which (a) is their own data, and (b) can be named anything they want anyway, so filtering on name seems pointless

Reply:
so way hon simply pass `false` to compile funtion?

Reply:
?